### PR TITLE
Add tests for the lookup method

### PR DESCRIPTION
**Pull request by ivyleavedtoadflax**
* Add test for the `lookup()` method.
* Tidy lookup method in `govukurllookup.py`.

### CODE REVIEW

File: govukurllookup.py [hunk 1]
```diff
@@ -31,36 +37,45 @@ def lookup(self):
...
+
+        # Lookup the url and return the json
+
+        results = requests.get(api_url).json()
+
+        # Check whether api returned a redirect, and if so look up the api_url
+        # using a standard http call so that we are returned the redirect url.
 
         if results['document_type'] == "redirect":
-            url = "https://www.gov.uk" + x
```
Comment (ellieking17) [inline govukurllookup.py]: i think would be more consistent if I use the {}.format here

File: govukurllookup.py [hunk 2]
```diff
@@ -31,36 +37,45 @@ def lookup(self):
...
-            s = requests.get(url)
-            redirected_url = s.url
-            redirected_api =redirected_url.replace("https://www.gov.uk", 
-                "https://www.gov.uk/api/content")
-            r = requests.get(redirected_api)
-            results = r.json()
+            redirect_url = "https://www.gov.uk" + url
```
Comment (ellieking17) [inline govukurllookup.py:62]: for consistency, I should probably use {}.format

File: requirements.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
 pandas==0.20.3
 beautifulsoup4==4.6.0
+requests==2.18.3
```
Comment (ellieking17) [inline requirements.txt:3]: That's why it was always missing!

**Review by ellieking17 — APPROVED** [review]
All lovely. How I enjoy this system.

File: govukurllookup.py (hunk 2 above)
Comment (ivyleavedtoadflax) [reply govukurllookup.py:62]: to be honest, there are so many ways to concatenate text in python. I read a blog somewhere that said that the `{}.format()` method is the one to use, but I couldn't tell you why!

File: requirements.txt (hunk 3 above)
Comment (ivyleavedtoadflax) [reply requirements.txt:3]: I've also added it to the setup.py file as a requirement before install the govukurllookup package, so we may not even need it here.

File: govukurllookup.py (hunk 2 above)
Comment (davidread) [reply govukurllookup.py:62]: `"" + ""` is marginally best here I think - it's clearest. `'{}'.format()` has more type checking, which is why it is often preferred, but only worry about that at beta stage. The other way is `'%s' % {}` which is the deprecated style.

**Pull request merged**